### DISCUSSION TITLE
Volume and battery indicator not working?

### DISCUSSION THREAD

**Discussion by Cotj73** [OnionUI/Onion#1081] [General]
I just got a Mini+ and installed Onion today.

I went through the Guide, updated the firmware to 0609, and after onion came up I found 2 issues. 

1) The volume was maxed out, and the side buttons do nothing
2) the battery indicator on the top right shows 0%

Note:
Iff I pull the SD card and load to the standard OS, it shows full battery and I can change the volume.

What do I need to do to fix these 2 issues?

**Comment by Cotj73** [comment]
I installed Onion 4.1.4

**Reply by thomasklemenc** [reply]
So you've installed this on your Miyoo Mini Plus?

![image](https://github.com/OnionUI/Onion/assets/6023076/f9048386-82c2-4054-9a65-9ff598ad429e)

**Comment by Cotj73** [comment]
Great
That is what I missed!

https://github.com/OnionUI/Onion/wiki/Installation

this pages needs to be updated because it states you should install 4.1.4 for Mini+

**Comment by Cotj73** [comment]
Nope I misread it!
